yarn.lock

```diff
@@ -2232,13 +2232,13 @@
     once "^1.4.0"
 
 "@octokit/request@^5.2.0":
-  version "5.3.4"
-  resolved "https://registry.yarnpkg.com/@octokit/request/-/request-5.3.4.tgz#fbc950bf785d59da3b0399fc6d042c8cf52e2905"
-  integrity sha512-qyj8G8BxQyXjt9Xu6NvfvOr1E0l35lsXtwm3SopsYg/JWXjlsnwqLc8rsD2OLguEL/JjLfBvrXr4az7z8Lch2A==
+  version "5.4.0"
+  resolved "https://registry.yarnpkg.com/@octokit/request/-/request-5.4.0.tgz#52382c830f0cf3295b5a03e308872ac0f5ccba9b"
+  integrity sha512-uAJO6GI8z8VHBqtY7VTL9iFy1Y+UTp5ShpI97tY5z0qBfYKE9rZCRsCm23VmF00x+IoNJ7a0nuVITs/+wS9/mg==
```

Comment:
I think pinning all the dependencies to a specific version, instead of a range, would keep this from happening to any of us (not something you caused @merelinguist)

Reply:
I would have thought the yarn.lock file should prevent carrotted events from upgrading automatically. Got a feeling that some kind of accidental dep upgrade possibly happened. Could the cli build have caused this?

Reply:
I think the problem was that I deleted yarn.lock while trying to figure out the prisma enoent problem. I’ve just reverted the file for safety.

Reply:
And yes, I think normally the lock file would prevented careted packages being upgraded randomly.